### ISSUE TITLE
LazyFrame sort `reverse` 

### ISSUE THREAD

**Issue by zba18** [xavctn/img2table#41]
Hi, just recently used your package and it's awesome. 

On a fresh install today, I noticed a breaking change. I think the param `reverse` should be swapped out with `descending`? [polars LazyFrame.sort](https://pola-rs.github.io/polars/py-polars/html/reference/lazyframe/api/polars.LazyFrame.sort.html)

```
lib/python3.11/site-packages/img2table/tables/processing/cells/deduplication.py:23, in deduplicate_cells_vertically(df_cells)
     16 df_cells = (df_cells.sort(by=["x1", "x2", "y1", "y2"])
     17             .with_columns(pl.lit(1).alias('ones'))
     18             .with_columns(pl.col('ones').cumsum().over(["x1", "x2", "y1"]).alias('cell_rk'))
     19             .filter(pl.col('cell_rk') == 1)
     20             )
     22 # Deduplicate on lower bound
---> 23 df_cells = (df_cells.sort(by=["x1", "x2", "y2", "y1"], reverse=[False, False, False, True])
     24             .with_columns(pl.lit(1).alias('ones'))
     25             .with_columns(pl.col('ones').cumsum().over(["x1", "x2", "y2"]).alias('cell_rk'))
     26             .filter(pl.col('cell_rk') == 1)
     27             )
     29 return df_cells.select(orig_cols)

TypeError: LazyFrame.sort() got an unexpected keyword argument 'reverse'
```

**Comment by xavctn** [comment]
Hi,
Thanks for the feedback, it must have been updated in a recent version, I will fix it.

**Comment by xavctn** [comment]
Nevermind,
I changed it in a recent release when polars released version 0.17 --> https://github.com/xavctn/img2table/blob/main/src/img2table/tables/processing/bordered_tables/cells/identification.py#L57.

From version 0.0.23 onwards, it shouldn't be an issue.

Can you check what is your installed version ?

**Comment by zba18** [comment]
Oh you're right! Some of my python dependencies led to me using 0.0.12 leading to this bug. Sorry to alarm you.
